src/OSPSuite.SimModelNative/src/GlobalConstants.cpp

```diff
@@ -13,7 +13,7 @@ const bool IsNan (const double d)
 #endif
 
 #ifdef linux
-	return isnan(d);
+	return std::isnan(d);
```

Comment:
s. https://stackoverflow.com/questions/33770374/why-is-isnan-ambiguous-and-how-to-avoid-it